### PR TITLE
qa/tasks/vstart_runner: fix sh to make it compatible with teth

### PR DESCRIPTION
**Pull request by lxbsz**
Fixes: https://tracker.ceph.com/issues/44555
Signed-off-by: Xiubo Li <xiubli@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/vstart_runner.py [hunk 1]
```diff
@@ -389,7 +389,9 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
 
         return proc
 
-    def sh(self, command, log_limit=1024, cwd=None, env=None):
+    def sh(self, command, log_limit=1024, cwd=None, env=None, **kwargs):
+        if type(command) is list:
```
Comment (tchaikov) [inline qa/tasks/vstart_runner.py]:
```suggestion
        if isinstance(command, list):
```

Comment (lxbsz) [reply qa/tasks/vstart_runner.py]: Fixed it.
Thanks

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-03-26_11:37:47-rados-wip-kefu-testing-2020-03-26-1712-distro-basic-smithi/

**Pull request merged**